internal/kuberesource/parts.go

```diff
@@ -325,6 +325,11 @@ func Coordinator(namespace string) *CoordinatorConfig {
 							).
 							WithResources(ResourceRequirements().
 								WithMemoryLimitAndRequest(100),
+							).
+							WithEnv(
+								NewEnvVar("CONTRAST_LOG_LEVEL", "debug"),
+								NewEnvVar("CONTRAST_LOG_FORMAT", "text"),
+								NewEnvVar("CONTRAST_LOG_SUBSYSTEMS", "*"),
```

Comment:
Is it safe to do this in production? Is there a way we can enable this only for dev builds?

Reply:
refactored it.